src/Components/PokeDetails/PokeDetails.js

```diff
@@ -1,23 +1,36 @@
-import React from 'react'
-import './PokeDetails.scss'
+import React from 'react';
+import './PokeDetails.scss';
 
-const PokeDetails = ({details}) => {
-  const pokemonName = details.name
-  const pokeFrontImage = details.sprites.front_default
+const PokeDetails = ({details, isShiny}) => {
+  const pokemonName = details.name;
+  const pokemonType = details.types.map(poke => poke.type.name + '\n');
+  const pokeFrontImage = details.sprites.front_default;
+  const pokeBackImage = details.sprites.back_default;
+  const pokeBackShiny = details.sprites.back_shiny;
+  const pokeFrontShiny = details.sprites.front_shiny;
+  console.log(details);
```

Comment:
could we de-structure some of these variables, or at least the sprites.

Reply:
done!